### PR TITLE
fix: default `wrangler tail` to pretty print

### PR DESCRIPTION
**Pull request by threepointone**
Fixes https://github.com/cloudflare/wrangler2/issues/707

### CODE REVIEW

File: packages/wrangler/src/tail/printing.ts [hunk 1]
```diff
@@ -24,13 +24,13 @@ export function prettyPrintLogs(data: WebSocket.RawData): void {
 
   if (eventMessage.logs.length > 0) {
     eventMessage.logs.forEach(({ level, message }) => {
-      console.log(`  (${level}) ${message}`);
+      console.log(`  (${level})`, message);
```
Comment (threepointone) [inline packages/wrangler/src/tail/printing.ts:27]: this fix prevents `[Object object]` being printed for complex objects

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
LGTM - please can you add tests to `tail.test.ts`.

**Comment by threepointone** [comment]
DOne

File: packages/wrangler/src/tail/printing.ts (hunk 1 above)
Comment (petebacondarwin) [reply packages/wrangler/src/tail/printing.ts:27]: This was the one I thought would most benefit from a test.

**Review by petebacondarwin — APPROVED** [review]
Cool.

**Comment by threepointone** [comment]
Oh damn I missed that. I'll add a test separately.

**Pull request merged**